app/src/main/java/eu/kanade/tachiyomi/ui/player/settings/sheets/subtitle/SubtitleFontPage.kt

```diff
@@ -111,7 +119,15 @@ private fun SubtitleFont(
             )
         }
 
+        OutlinedTextField(
```

Comment:
Shouldn't we make this a dropdown menu?
If we already parse the folder for fonts then we can show the user which ones are available...